README.md

```diff
@@ -1,70 +1,77 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Space Travelers' Hub
 
-## Available Scripts
+> Countries Catalog is a Single Page App built with react.It fetches data from https://restcountries.com/ api and shows all the countries of the world. It also show their population on homepage and upon clicking on any country's card, it takes to the detail page and shows more info about that specific country.In this web app I used Redux as a state management library. And for testing, I used react-testing-library with Jest.
 
-In the project directory, you can run:
+# Screenshot
 
-### `npm start`
```

Comment:
- [ ] Please use this new Microverse README [template ](https://github.com/microverseinc/readme-template), It will make the README file looks more professional.